lib/Service/SessionService.php

```diff
@@ -228,6 +228,6 @@ private function getColorForGuestName(string $guestName = null): string {
 		$guestName = $this->userId ?? $guestName;
 		$uniqueGuestId = !empty($guestName) ? $guestName : $this->secureRandom->generate(12);
 		$color = $this->avatarManager->getGuestAvatar($uniqueGuestId)->avatarBackgroundColor($uniqueGuestId);
-		return sprintf("#%02x%02x%02x", $color->r, $color->g, $color->b);
+		return sprintf("#%02x%02x%02x", $color->red(), $color->green(), $color->blue());
```

Comment:
As now available with the new API:

```suggestion
		return $color->name();
```

Reply:
Awesome!